docs/modules/node.md

```diff
@@ -612,6 +612,25 @@ The random number in the appropriate range. Note that the zero argument form wil
 print ("I rolled a", node.random(6))
 ```
 
+## node.ccount()
+
+Get value of CPU CCOUNT register which contains CPU ticks. It support CPU80 and CPU160.
+
+This allows for calculation of elapsed time with nanosecond precision. For example for CPU80 there are 80 ticks/us (80000 ticks/ms).
```

Comment:
It isn't really nanosecond precision -- you get the number of clock cycles -- at roughly 80/us or 160/us. Any timing accuracy is dependant on the accuracy of the oscillator which has errors in (as I recall) 100s of ppm.

Reply:
it isn't of course, i must have been brain dead ;) corrected to microseconds.